src/components/AccordionSection/AccordionSection.tsx

```diff
@@ -84,6 +85,7 @@ AccordionSection.propTypes = {
   setExpanded: PropTypes.func,
   title: PropTypes.string,
   titleElement: PropTypes.oneOf(["h2", "h3", "h4", "h5", "h6"]),
+  headingLevel: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
```

Comment:
The `titleElement` prop on the line above doesn't accept `h1`, and it _probably_ is reasonable that an accordion title wouldn't be the most important heading on the page
```suggestion
  headingLevel: PropTypes.oneOf([2, 3, 4, 5, 6]),
```

Reply:
Good catch, thanks.